Assets/HoloToolkit/SpatialSound/Scripts/IAudioInfluencer.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Interface that is implemented by any class that wishes to influence how an audio source sounds.
+    /// </summary>
+    public interface IAudioInfluencer
+    {
+        /// <summary>
+        /// Applies an audio effect.
+        /// </summary>
+        /// <param name="emitter">The GameObject on which the effect is to be applied.</param>
+        /// <param name="audioSource">The AudioSource that will be impacted by the effect.</param>
+        void ApplyEffect(GameObject emitter,
```

Comment:
There is no requirement for an influencer to use a single filter. Returning a GameObject would be insufficient.